### ISSUE TITLE
no-emphasis-as-heading bug

### ISSUE THREAD

**Issue by RyanZim** [wooorm/remark-lint#139]
Originally reported here: https://github.com/sindresorhus/awesome-lint/issues/14

The `no-emphasis-as-heading` rule doesn't like this:

> # Awesome npm scripts [![Awesome](https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg)](https://github.com/sindresorhus/awesome)
> [![Travis branch](https://img.shields.io/travis/RyanZim/awesome-npm-scripts/master.svg?maxAge=2592000?style=plastic)](https://travis-ci.org/RyanZim/awesome-npm-scripts)
>
> > Everything awesome related to npm scripts and using npm as a build tool.
>
> *You might also like [awesome-npm](https://github.com/sindresorhus/awesome-npm).*
> 
> **Notice: I'm currently too busy to actively expand this list; therefore, I've decided to make this an [OPEN Open Source Project](http://openopensource.org). Individuals making significant and valuable contributions are given commit-access to the project to contribute as they see fit.**
> 
> ## Contents
> ...

https://github.com/RyanZim/awesome-npm-scripts

The emphasis has nothing to do with the heading; it's just a bold notice that happens to come right before the top heading.

**Comment by wooorm** [comment]
Hiya Ryan, thanks for taking the time to write this issue! 👋

First: sort-of unrelated to this issue, but the first part of that readme is pretty funky: there’s a block quote (to emphasise the description), then there italics (to emphasise a similar project), then there’s bold (to emphasise a warning). Why emphasise everything? Doesn’t that kind-off defeat the purpose of emphasis? :)

Second: this rule, `no-emphasis-as-heading`, checks that no paragraph is completely emphasised. Take this code:

```markdown
*Foo*

Bar.

__Qux__

Quux.
```

Those two should probably be headings! What do you propose as a solution, where my example still warns, where yours does not?

Note: there’s also a way to ignore this message: `<!--lint disable no-emphasis-as-heading-->`

And finally, you could rewrite your emphasis like this: `**Related**: You might also like awesome-npm.`, while still drawing some attention to that paragraph.

**Comment by RyanZim** [comment]
As a workaround, I removed the emphasis from "You might also like awesome-npm."
